docs/sql_reference/src/asciidoc/_chapters/sql_statements.adoc

```diff
@@ -6642,6 +6642,7 @@ sql-query is:
   | query-expr-and-order
 
 query-specification is:
+with-clause

```

Comment:
There should be an square bracket around the with-clause, since it is optional. Others look great to me.
[ with-clause ]